File: JavaScript_Basic_04_2023_SolutionsOfProblems/03.Ex_PB-JS-Nested-Conditional-Statements/P07_HotelRoom.js
function hotelRoom(input) {
    let monthName = input[0];
    let stayNumber = Number(input[1]);

    let totalCostStudio = 0.0;
    let totalCostApartment = 0.0;

    switch (monthName) {
        case "May":
        case "October":
            totalCostApartment = stayNumber * 65;
            totalCostStudio = stayNumber * 50;
            if (stayNumber > 14) {
                totalCostStudio = totalCostStudio * 0.7;
                totalCostApartment = totalCostApartment * 0.9;
            } else if (stayNumber > 7) {
                totalCostStudio = totalCostStudio * 0.95;
            }
            break;
        case "June":
        case "September":
            totalCostApartment = stayNumber * 68.7;
            totalCostStudio = stayNumber * 75.2;
            if (stayNumber > 14) {
                totalCostStudio = totalCostStudio * 0.8;
                totalCostApartment = totalCostApartment * 0.9;
            }
            break;
        case "July":
        case "August":
            totalCostApartment = stayNumber * 77;
            totalCostStudio = stayNumber * 76;
            if (stayNumber > 14) {
                totalCostApartment = totalCostApartment * 0.9;
            }
            break;
    }
    console.log(`Apartment: ${totalCostApartment.toFixed(2)} lv.`);
    console.log(`Studio: ${totalCostStudio.toFixed(2)} lv.`);
}

hotelRoom(["August",
    "20"])

File: Java_Basic_Jan_2023_SolutionsOfProblems/P03Ex_ConditionalStatements_Advanced/P07_HotelRoom.java
package P03Ex_ConditionalStatements_Advanced;

import java.util.Scanner;

public class P07_HotelRoom {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double priceStudio = 0;
        double priceApartment = 0;

        String mount = scanner.nextLine();
        int numberStays = Integer.parseInt(scanner.nextLine());

        switch (mount) {
            case "May":
            case "October":
                priceStudio = numberStays * 50.0;
                priceApartment = numberStays * 65.0;
                if (numberStays > 14) {
                    priceStudio = priceStudio * 0.7;
                    priceApartment = priceApartment * 0.9;
                } else if (numberStays > 7) {
                    priceStudio = priceStudio * 0.95;
                }
                break;
            case "June":
            case "September":
                priceStudio = numberStays * 75.20;
                priceApartment = numberStays * 68.70;
                if (numberStays > 14) {
                    priceStudio = priceStudio * 0.8;
                    priceApartment = priceApartment * 0.9;
                }
                break;
            case "July":
            case "August":
                priceStudio = numberStays * 76;
                priceApartment = numberStays * 77;
                if (numberStays > 14) {
                    priceApartment = priceApartment * 0.9;
                }
                break;
        }
        System.out.printf("Apartment: %.2f lv.%n", priceApartment);
        System.out.printf("Studio: %.2f lv.", priceStudio);
    }
}
